doc/md/fundamentals/2-types/11-stable-types.md

```diff
@@ -4,7 +4,7 @@ sidebar_position: 11
 
 # Stable types
 
-Recall that [shared types](https://internetcomputer.org/docs/motoko/fundamentals/types/shared-types) can be transmitted to other actors and canisters as arguments and or be the result of shared functions. 
+Recall that [shared types](https://internetcomputer.org/docs/motoko/fundamentals/types/shared-types) can be transmitted to other actors and canisters as arguments and or be the result of shared functions.
 
 **Stable types** include all shared types and are the types of values that can be stored in Motoko's stable variables. Storing a value in a stable variable ensures the value can be preserved across upgrades of the actor, allowing an application to preserve state without requiring a file system or database. The ability to safely and easily preserve data across upgrades is one the distinguishing features of Motoko that separates it from other mainstream languages.
 
```

Comment:
In the table below, may point out that (non-shared) function and futures (async(*) T) are neither shared nor stable.

Reply:
In the table, Also, the T in [T], [var T], ?T must itself be stable or shared. Maybe you can add a footnote.

Eg.

| Immutable arrays (`[T]`)     | Yes* | Yes** |

* provided T stable
** provided T shared.

Reply:
Maybe add a line to the table for regions (Yes | No)